server/kadalu-quotad/quotad.py

```diff
@@ -106,9 +109,20 @@ def start():
     """
     first_time = True
     while True:
-        crawl()
-        time.sleep(1)
-        if not first_time:
+        brick_path = os.environ["BRICK_PATH"]
+        if brick_path:
+            crawl(brick_path)
+        try:
+            with open(CONFIG_FILE) as f:
+                for brick in f.readlines():
```

Comment:
@aravindavk made the crawl based on config file for now here. Isn't this enough?

Reply:
Sorry I missed this part. This is good